test/e2e/specs/updates/keyrotation/keyrotation.go

```diff
@@ -0,0 +1,147 @@
+package keyrotation
+
+import (
+	"context"
+	"flag"
+	"io/ioutil"
+	"os"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	v20180930preview "github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	"github.com/openshift/openshift-azure/pkg/config"
+	"github.com/openshift/openshift-azure/pkg/fakerp"
+	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+)
+
+var _ = Describe("Key Rotation E2E tests [KeyRotation] [Update]", func() {
+	defer GinkgoRecover()
```

Comment:
I don't believe this should be necessary, can it be removed?

Reply:
alright